SDL/Kernels.h

```diff
@@ -18,12 +18,12 @@ namespace SDL {
 
   ALPAKA_FN_ACC ALPAKA_FN_INLINE void rmPixelTripletFromMemory(struct SDL::pixelTriplets& pixelTripletsInGPU,
                                                                unsigned int pixelTripletIndex) {
-    pixelTripletsInGPU.isDup[pixelTripletIndex] = 1;
+    pixelTripletsInGPU.isDup[pixelTripletIndex] = true;
```

Comment:
Out of curiosity, why this change? Did it cause an issue? Is this defined as a bool variable?

Reply:
this was from a clang check; a part missed in #356